tutorials/haskell/haskell-time-tutorial/tutorial.md

```diff
@@ -0,0 +1,537 @@
+---
+title: Time, patterns and a little more
+published: 2021-11-25
+tags: haskell, pattern-synonyms, property-testing
+libraries: time
+language: haskell
+author: Felix Miño
+author-name: Felix Miño
+github-profile: felixminom
+description: In this tutorial we’re going to explore the use of pattern synonyms in Haskell, later we’ll give a brief overview to the Stack Builders’ contribution to the `time` Haskell library and finally we’ll review how to use the utilities that were introduced in the contribution.
+---
+
+As Stack Builders one of the core values of our company is contributing
+to open source (OSS). We believe that we have the power to change
+people’s lives (or at least make it easier) by pushing the boundaries
+of the software industry and we’re happy to announce that the past month
+we were able to contribute to one of Haskell's core libraries,
+[time][time-hackage].
+
+We started with the idea: “What if we bring some utilities that Ruby on
+Rails (RoR) has to Haskell?”. For instance, “How can I get the range of
+days that a month has?” or “How can I get a quarter’s boundaries?” were
+some of the questions that came to our mind. In the following example
+I will show the Haskell code needed to get all days in a month and its
+version in RoR.
```

Comment:
Since the order changed (from Haskell then Ruby on Rails to Ruby on Rails then Haskell), can you make it clearer that the first one is Ruby on Rails? Here's a possible way to make that clearer:

```suggestion
I will show the RoR code to get all days in a month and then a Haskell
version to do so
```